### DISCUSSION TITLE
Equation SoC

### DISCUSSION THREAD

**Discussion by SyarifulAzis1018** [pybamm-team/PyBaMM#2280] [Q&A] [answered]
Can you show to me, about equation SoC and Terminal Voltage that you use in pybamm? and where the code can i found it?

**Comment by brosaplanella** [comment, marked as answer]
The terminal voltage depends on the model you are using, if you go to the docs you can find links to the relevant articles with the models. The SoC is not defined as is, but you can access the dimensionless concentrations in the electrodes a use them to define the SoC.

**Reply by SyarifulAzis1018** [reply]
So, can i count SoC from `Discharge capacity [A.h]` ?

**Reply by brosaplanella** [reply]
Yes, you can use the discharge capacity and do Coulomb counting

**Reply by SyarifulAzis1018** [reply]
Okey, Thank you very much
